das_client/test_resources/sp/SFERA_SP_9999_4.xml

```diff
@@ -34,7 +34,7 @@
                 <NetworkSpecificParameter name="mainStation" value="CH9994"/>
             </TAF_TAP_Location_NSP>
         </TAF_TAP_Location>
-        <NetworkSpecificArea startEndQualifier="Ends" startLocation="800" endLocation="1" name="nonStandardTrackEquipment">
+        <NetworkSpecificArea startEndQualifier="Ends" endLocation="800" name="nonStandardTrackEquipment">
```

Comment:
Das selbe hier für:
`startEndQualifier="Ends" startLocation="800" endLocation="1"`

Reply:
das ist glaub code, den du gemacht hast. Kannst du das bei dir korrekt machen?

Reply:
@Grodien Das Dito war auf [diesen Kommentar](https://github.com/SchweizerischeBundesbahnen/DAS/pull/413#discussion_r1858904646) bezogen. Da du das TODO entfernt hast, sollte das in dem Fall auch stimmen 👍